packages/react-native/React/Base/RCTBridgeProxy.mm

```diff
@@ -75,10 +78,10 @@ - (Class)executorClass
  * Used By:
  *  - RCTBlobCollector
  */
-- (jsi::Runtime *)runtime
+- (void *)runtime
 {
-  [self logWarning:@"This method is unsupported. Returning nullptr." cmd:_cmd];
-  return nullptr;
+  [self logWarning:@"Please migrate to C++ TurboModule or RuntimeExecutor." cmd:_cmd];
```

Comment:
Are there any docs for `RuntimeExecutor`? How would one create a native C++ module that has access to `jsi::Runtime` and gets auto-linked/initialized properly on Android and iOS?